verification.py

```diff
@@ -160,27 +160,24 @@ def calculate_accuracy(threshold, dist, actual_issame, useCos=False):
 
 
 def calculate_val(thresholds, embeddings1, embeddings2, actual_issame, far_target, nrof_folds=10):
-    '''
-    Copy from [insightface](https://github.com/deepinsight/insightface)
-    :param thresholds:
-    :param embeddings1:
-    :param embeddings2:
-    :param actual_issame:
-    :param far_target:
-    :param nrof_folds:
-    :return:
-    '''
     assert (embeddings1.shape[0] == embeddings2.shape[0])
     assert (embeddings1.shape[1] == embeddings2.shape[1])
-    nrof_pairs = min(len(actual_issame), embeddings1.shape[0])
+    diff = np.subtract(embeddings1, embeddings2)
+    dist = np.sum(np.square(diff), 1)
+    return calculate_val(thresholds, dist, actual_issame, far_target, nrof_folds=nrof_folds)
```

Comment:
This line should be 
```suggestion
    return calculate_val_by_diff(thresholds, dist, actual_issame, far_target, nrof_folds=nrof_folds)
```
?